robottelo.properties.sample

```diff
@@ -106,7 +106,9 @@ ssh_key=
 # be pulled from there instead of using the CDN channel. These information is
 # more suited to be used for downstream, downstream-iso and zstream builds.
 # capsule_repo=http://capsule/repo
-# sattools_repo=http://sattools/repo
+# sattools_repo=
```

Comment:
Maybe this should be two separated options, like we have above the rhel6_os and rhel7_os. Mostly all the options that needs both systems they are separated so I was wondering if we should follow that for these too.